.scalasteward.conf

```
@@ -0,0 +1 @@
+pullRequests.frequency = "0 0 1,15 * *"
```

Unfortunately the library for cron expressions used by Scala Steward won't accept that string. If "day of month" is specified, "day of week" needs to be `?`:
```suggestion
pullRequests.frequency = "0 0 1,15 * ?"
```

---

Ahh, ok. Thanks! I switched it.